features/git-town-config/setup.feature

```diff
@@ -11,83 +11,9 @@ Feature: Initial configuration
 
 
   Scenario: succeeds on valid main branch and perennial branch names
-    When I run `git-town config --setup` and enter:
-      | main       |
-      | production |
-      | dev        |
-      |            |
+    When I run `git-town config --setup` and answer the prompts:
+      | PROMPT                                     | ANSWER                    |
+      | Please specify the main development branch | DOWN, ENTER               |
```

Comment:
What do you think about distinguishing pressed keys better from text that is entered? What I often see is something like this:

```
[DOWN] [ENTER]
```

Alternatively, we could use unicode:

```
⬇️ ⏎
```

Reply:
Updated to `[DOWN][ENTER]`